modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -700,6 +726,8 @@ public CompletableFuture<Void> stopAsync() {
 
         initialClusterConfigurationFuture.completeExceptionally(new NodeStoppingException());
 
+        fireEvent(ClusterManagerGroupEvent.AFTER_STOP_RAFT_GROUP, EmptyEventParameters.INSTANCE);
```

Comment:
Do we need to return this future from the method? So that the component would not stop until all listeners finish their work.

Reply:
Yeah this was a bug